.github/workflows/deploy-on-version-tag.yml

```diff
@@ -0,0 +1,57 @@
+name: Deployment
+
+on:
+  push:
+    tags:
+      - "20.*"
+      - "21.*"
+      - "22.*"
+      - "23.*"
+      - "24.*"
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - run: npm install
+    - run: npm test
+    - run: du -a # Log all available files # TODO: maybe remove
+    - name: Version and tag
+      run: |
+        export VER=$(echo "$GITHUB_REF" | sed -e 's/refs\/heads\///')
+        echo https://github.com/$GITHUB_REPOSITORY/tree/$VER > distribution/SOURCE_URL
+        npm run version
+    - uses: actions/upload-artifact@v1
+      with:
+        name: built-extension
+        path: distribution
+
+  deploy-to-chrome:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/download-artifact@v1
+      with:
+        name: built-extension
+        path: distribution
+    - run: cd distribution && echo npx chrome-webstore-upload-cli upload --auto-publish # TODO: drop `echo` before merging
+      env:
+        EXTENSION_ID: ${{ secrets.EXTENSION_ID }}
+        CLIENT_ID: ${{ secrets.CLIENT_ID }}
+        CLIENT_SECRET: ${{ secrets.CLIENT_SECRET }}
+        REFRESH_TOKEN: ${{ secrets.REFRESH_TOKEN }}
+
+  deploy-to-firefox:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/download-artifact@v1
+      with:
+        name: built-extension
+        path: distribution
+    - run: cd distribution && echo npx web-ext-submit # TODO: drop `echo` before merging
+      env:
+        WEB_EXT_API_KEY: ${{ secrets.WEB_EXT_API_KEY }}
+        WEB_EXT_API_SECRET: ${{ secrets.WEB_EXT_API_SECRET }}
```

Comment:
If you set these ENVs on Refined GitHub, we can drop Travis deployment (and its issues/quirks)